css/courses.css

```diff
@@ -0,0 +1,97 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800&display=swap");
+body{
+    background-color: white;
+}
+
+
+/* Styling for the header */
+header{
+    margin-top:150px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5673604)